Source/WebCore/page/MemoryRelease.cpp

```diff
@@ -197,6 +197,7 @@ void logMemoryStatistics(LogMemoryStatisticsReason reason)
     const char* description = logMemoryStatisticsReasonDescription(reason);
 
     RELEASE_LOG(MemoryPressure, "WebKit memory usage statistics at time of %" PUBLIC_LOG_STRING ":", description);
+    RELEASE_LOG(MemoryPressure, "Websam state: %" PUBLIC_LOG_STRING, MemoryPressureHandler::processStateDescription());
```

Comment:
`MemoryPressureHandler::processStateDescription().characters()` once you switch to ASCIILiteral.